src/Makefile

```diff
@@ -286,12 +310,21 @@ ifneq (,$(filter ${DEBUGAD},YES yes y Y 1))
 	DEFINES+=-DDEBUGAD
 endif
 
-ifneq (,$(filter ${DEBUG},YES yes y Y 1))
+# Debug symbols
+ifeq (${DBG_SYM_SUPPORT}, yes)
 	OSSEC_CFLAGS+=-g
 else
-	OSSEC_CFLAGS+=-DNDEBUG
+ifneq (,$(filter ${DEBUG},YES yes y Y 1))
+	OSSEC_CFLAGS+=-g
```

Comment:
The OSSEC_CFLAGS flag with option DNDEBUG won't use anymore?

Reply:
Solved! NDEBUG definition will only happen when DEBUG is not set.